terraform/ses/provision/main.tf

```diff
@@ -22,7 +27,7 @@ resource "aws_ses_domain_identity" "identity" {
 
 resource "aws_route53_record" "record" {
   zone_id = data.aws_route53_zone.zone.zone_id
-  name    = "_amazonses.${aws_ses_domain_identity.identity.id}"
+  name    = "${local.instance_id}.${aws_ses_domain_identity.identity.id}"
```

Comment:
It turns out that the "_amazonses." prefix is not disposable... Need to fix this line to include it again!